### DISCUSSION TITLE
Back-off restarting failed container [May 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2190] [Discord Threads]
> 🙋‍♂️ **cirsten**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### can u run `kubectl describe pod [name]` to check the message  
#  
> 🙋‍♂️ **cirsten**  
#### Do you mean in shell, of so, I get command not found  
#  
> 🙋‍♂️ **cirsten**  
#### If so  
#  
> 👨‍🚀 **wingkwong**  
#### im not familiar with truecharts. is there any place to check logs?  
#  
> 👨‍🚀 **wingkwong**  
#### besides, can u share some steps how u installed?  
#  
> 🙋‍♂️ **cirsten**  
#### True charts is a collection of installable app for the Truenas server software   
#### It takes away the need for programming or command line access to install software   
####    
#### So I simply select nocodb from the list of apps and select install   
####    
#### I assumed it was created by some one from NocoDB to help people like me   
####    
#### I'll see if I call get the logs  
#  
> 🙋‍♂️ **cirsten**  
#### 2022-05-19T14:29:16.314670726Z [38;5;6mredis [38;5;5m14:29:16.24 [0m   
#### 2022-05-19T14:29:16.317456622Z [38;5;6mredis [38;5;5m14:29:16.31 [0m[1mWelcome to the Bitnami redis container[0m   
#### 2022-05-19T14:29:16.320177729Z [38;5;6mredis [38;5;5m14:29:16.31 [0mSubscribe to project updates by watching [1mhttps://github.com/bitnami/bitnami-docker-redis[0m   
#### 2022-05-19T14:29:16.322194564Z [38;5;6mredis [38;5;5m14:29:16.32 [0mSubmit issues and feature requests at [1mhttps://github.com/bitnami/bitnami-docker-redis/issues[0m   
#### 2022-05-19T14:29:16.324799124Z [38;5;6mredis [38;5;5m14:29:16.32 [0m   
#### 2022-05-19T14:29:16.327122895Z [38;5;6mredis [38;5;5m14:29:16.32 [0m[38;5;2mINFO [0m ==> ** Starting Redis setup **   
#### 2022-05-19T14:29:16.358009726Z [38;5;6mredis [38;5;5m14:29:16.35 [0m[38;5;3mWARN [0m ==> You set the environment variable ALLOW_EMPTY_PASSWORD=yes. For safety reasons, do not use this flag in a production environment.   
#### 2022-05-19T14:29:16.432542896Z [38;5;6mredis [38;5;5m14:29:16.43 [0m[38;5;2mINFO [0m ==> Initializing Redis   
#### 2022-05-19T14:29:16.447652298Z mkdir: cannot create directory '/opt/bitnami/redis/tmp': Permission denied  
#  
> 🙋‍♂️ **cirsten**  
#### 2022-05-19T14:29:49.110522901Z [38;5;6mpostgresql [38;5;5m14:29:49.10 [0m   
#### 2022-05-19T14:29:49.112258746Z [38;5;6mpostgresql [38;5;5m14:29:49.11 [0m[1mWelcome to the Bitnami postgresql container[0m   
#### 2022-05-19T14:29:49.114051448Z [38;5;6mpostgresql [38;5;5m14:29:49.11 [0mSubscribe to project updates by watching [1mhttps://github.com/bitnami/bitnami-docker-postgresql[0m   
#### 2022-05-19T14:29:49.115865976Z [38;5;6mpostgresql [38;5;5m14:29:49.11 [0mSubmit issues and feature requests at [1mhttps://github.com/bitnami/bitnami-docker-postgresql/issues[0m   
#### 2022-05-19T14:29:49.117614653Z [38;5;6mpostgresql [38;5;5m14:29:49.11 [0m   
#### 2022-05-19T14:29:49.208304249Z [38;5;6mpostgresql [38;5;5m14:29:49.20 [0m[38;5;2mINFO [0m ==> ** Starting PostgreSQL setup **   
#### 2022-05-19T14:29:49.244218696Z [38;5;6mpostgresql [38;5;5m14:29:49.24 [0m[38;5;2mINFO [0m ==> Validating settings in POSTGRESQL_* env vars..   
#### 2022-05-19T14:29:49.252692390Z [38;5;6mpostgresql [38;5;5m14:29:49.25 [0m[38;5;2mINFO [0m ==> Loading custom pre-init scripts...   
#### 2022-05-19T14:29:49.444510948Z [38;5;6mpostgresql [38;5;5m14:29:49.44 [0m[38;5;2mINFO [0m ==> Initializing PostgreSQL database...   
#### 2022-05-19T14:29:49.487644679Z [38;5;6mpostgresql [38;5;5m14:29:49.48 [0m[38;5;2mINFO [0m ==> pg_hba.conf file not detected. Generating it...   
#### 2022-05-19T14:29:49.490393656Z [38;5;6mpostgresql [38;5;5m14:29:49.48 [0m[38;5;2mINFO [0m ==> Generating local authentication configuration   
#### 2022-05-19T14:29:49.491563889Z /opt/bitnami/scripts/libpostgresql.sh: line 222: /opt/bitnami/postgresql/conf/pg_hba.conf: Permission denied   
#### 2022-05-19T14:29:49.539583755Z /opt/bitnami/scripts/libfile.sh: line 39: /opt/bitnami/postgresql/conf/postgresql.conf: Permission denied  
#  
> 👨‍🚀 **wingkwong**  
#### seems something wrong with permission on your server  
#  
> 🙋‍♂️ **cirsten**  
#### Yep see that (now),0not sure how to change it though   
####    
#### I'm off to bed now.   
####    
#### I'll trawl through truenas,/truechart forums again and see what I can find   
####    
#### Thanks for your help  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/976820757856407592)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
